src/mixins/bcol-mixin.ts

```diff
@@ -4,24 +4,28 @@ import axios from '@/utils/axios-auth'
 @Component({})
 export default class BcolMixin extends Vue {
   async getErrorObj (errCode: string): Promise<any> {
-    const fetchUrl: string = this.payApi + 'codes/errors/' + errCode
-    // Currently no desirable way to handle errors during this request,
-    // null is returned in any error situation regardless.
-    const errObj: any = await axios.get(fetchUrl).catch()
-    if (errObj?.data) {
-      return errObj.data
+    try {
+      const fetchUrl: string = this.payApi + 'codes/errors/' + errCode
+      // Currently no desirable way to handle errors during this request,
+      // null is returned in any error situation regardless.
+      const errObj: any = await axios.get(fetchUrl).catch()
+      if (errObj?.data) {
+        return errObj.data
+      }
+      return null
+    } catch (error) {
+      return null
```

Comment:
Just curious if you need the try catch if `null` is return regardless. Are you able to `return errObj?.data || null` or are you expecting a potential error on the `fetchUrl`?

Reply:
I'm wondering the same thing.
Also, if you get a lint error about "error not used" then you can change this to the special form `catch (e)` (where it ignores the fact that you're not using e).